.github/workflows/release-web.yml

```diff
@@ -0,0 +1,111 @@
+name: Build and deploy web
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*\+*'
+
+jobs:
+  extract-version:
```

Comment:
Maybe we could group this workflow with the previous one, so that this job isn't declared and executed several times.